packages/components/src/components/Dialog/Dialog.tsx

```diff
@@ -0,0 +1,194 @@
+    {
+        maxWidth = '800px',
+        minWidth = '400px',
+        minHeight = '200px',
+        padding = 'compact',
+        'data-test-id': dataTestId = 'fondue-dialog-content',
+        showUnderlay = false,
+        children,
+        ...props
+    }: DialogContentProps,
+    ref: ForwardedRef<HTMLDivElement>,
+) => {
+    return (
+        <RadixDialog.Portal>
+            <RadixDialog.Overlay className={dialogUnderlayStyles({ showUnderlay })} />
+            <RadixDialog.Content
+                style={
+                    {
+                        '--dialog-max-width': maxWidth,
+                        '--dialog-min-width': minWidth,
+                        '--dialog-min-height': minHeight,
+                    } as CSSProperties
+                }
+                ref={ref}
+                className={dialogContentStyles({ ...props })}
+                data-dialog-spacing={padding}
+                data-test-id={dataTestId}
+                {...props}
+            >
+                {children}
+            </RadixDialog.Content>
+        </RadixDialog.Portal>
+    );
+};
+DialogContent.displayName = 'Dialog.Content';
+
+export const DialogHeader = (
+    { children, 'data-test-id': dataTestId = 'fondue-dialog-header' }: DialogHeaderProps,
+    ref: ForwardedRef<HTMLDivElement>,
+) => {
+    return (
+        <div data-test-id={dataTestId} ref={ref} className={dialogHeaderStyles} data-dialog-layout-component>
+            <div>{children}</div>
+            <RadixDialog.Close role="button" data-test-id={`${dataTestId}-close`} className="tw-cursor-pointer">
+                <IconCross size={20} />
+            </RadixDialog.Close>
+        </div>
+    );
+};
+DialogHeader.displayName = 'Dialog.Header';
+
+export const DialogFooter = (
+    { children, 'data-test-id': dataTestId = 'fondue-dialog-footer' }: DialogFooterProps,
+    ref: ForwardedRef<HTMLDivElement>,
+) => {
+    return (
+        <div data-test-id={dataTestId} ref={ref} className={dialogFooterStyles} data-dialog-layout-component>
+            {children}
+        </div>
+    );
+};
+DialogFooter.displayName = 'Dialog.Footer';
+
+export const DialogBody = (
+    { children, 'data-test-id': dataTestId = 'fondue-dialog-body' }: DialogBodyProps,
+    ref: ForwardedRef<HTMLDivElement>,
+) => {
+    return (
+        <div data-test-id={dataTestId} ref={ref} className={dialogBodyStyles} data-dialog-layout-component>
+            {children}
+        </div>
+    );
+};
+DialogBody.displayName = 'Dialog.Body';
+
+export const SideContent = (
+    { children, 'data-test-id': dataTestId = 'fondue-dialog-side-content' }: DialogBodyProps,
+    ref: ForwardedRef<HTMLDivElement>,
+) => {
+    return (
+        <div data-test-id={dataTestId} ref={ref} className={dialogSideContentStyles} data-dialog-layout-component>
+            {children}
+        </div>
+    );
+};
+SideContent.displayName = 'Dialog.SideContent';
+
+export const Dialog = {
+    Root: DialogRoot,
```

Comment:
why not forward root?

Reply:
Doe not render a element per se and is not supported by radix